functional-tests/plege_sector_test.go

```diff
@@ -24,6 +24,7 @@ func TestMiningPledgeSector(t *testing.T) {
 
 	genTime := int64(1000000000)
 	blockTime := 1 * time.Second
+	propDelay := 200 * time.Millisecond
```

Comment:
Hmm, this smells flaky. Given the fake clock I would be much happier if this were set to zero but everything still worked, meaning that the MineOnce underneath here didn't return until the block had been consumed and synced by the local node.

I guess we don't quite have the hookups to achieve that yet.